app/server/me_api/organisation.py

```diff
@@ -0,0 +1,23 @@
+from flask import g, make_response, jsonify
+from flask.views import MethodView
+
+from server.schemas import me_organisation_schema
+from server.utils.auth import requires_auth
+
+
+class MeOrganisationAPI(MethodView):
+    @requires_auth
+    def get(self):
+
+        organisation = g.user.default_organisation
```

Comment:
do we want to use default organisation?

Reply:
shouldn't this take the active organisation defined by the query string?

Reply:
so every request now passes in org in the query string? always as 'org'?